test/functional-portable/corerp/noncloud/cli/cli_test.go

```diff
@@ -59,21 +59,21 @@ const (
 	retries = 10
 )
 
-func verifyRecipeCLI(ctx context.Context, t *testing.T, test shared.RPTest) {
-	options := shared.NewRPTestOptions(t)
+func verifyRecipeCLI(ctx context.Context, t *testing.T, test corerp.RPTest) {
```

Comment:
Is there a reason why this is part of corerp? Isn't this a CLI test?

Reply:
This will go away once the cli changes PR gets merged